test/campfire_social_hour/zoom_credential_agent_test.exs

```diff
@@ -0,0 +1,16 @@
+defmodule CampfireSocialHour.ZoomCredentialAgentTest do
+  use CampfireSocialHour.ZoomApiCase, async: false
+
+  alias CampfireSocialHour.ZoomCredentialAgent
+
+  describe "access_token/0" do
+    test "when zoom api returns a valid response" do
+      assert ZoomCredentialAgent.access_token() == "abc123"
```

Comment:
as mentioned in the previous comment, the best way to test genservers in isolation from the one started in supervision tree is to allow setting its name and then explicitly starting it and interacting with _it_ in the test:

```suggestion
    test "when zoom api returns a valid response", %{test: test} do
      start_supervised!({ZoomCredentialAgent, name: test})
      assert ZoomCredentialAgent.access_token(test) == "abc123"
```